applications/RANSApplication/python_scripts/rans_analysis.py

```diff
@@ -0,0 +1,59 @@
+from sys import argv
+
+import KratosMultiphysics as Kratos
+
+from KratosMultiphysics.analysis_stage import AnalysisStage
+from KratosMultiphysics.RANSApplication.coupled_rans_solver import CoupledRANSSolver
+
+class RANSAnalysis(AnalysisStage):
```

Comment:
I think we should derive this from the `FluidDynamicsAnalysis`. Aside of avoiding the duplication of some methods, this will ensure that the processes order is always the same in both applications.

Reply:
Agreed, and changed locally. Will push once you finish review :)